csd/aux/registry.envsubst.yaml

```diff
@@ -47,7 +47,18 @@ server:
 
 # Logging settings.
 logging:
-  level: INFO
+  level: ${HWX_REGISTRY_LOG_THRESHOLD}
   loggers:
-    com.hortonworks.registries: INFO
-
+    com.hortonworks.registries: ${HWX_REGISTRY_LOG_THRESHOLD}
+  appenders:
+    - type: console
+      threshold: ${HWX_REGISTRY_LOG_THRESHOLD}
+      logFormat: "${HWX_REGISTRY_LOG_CONSOLE_FORMAT}"
+    - type: file
+      currentLogFilename: ${HWX_REGISTRY_LOG_DIR}/${HWX_REGISTRY_LOG_FILE}
+      threshold: ${HWX_REGISTRY_LOG_THRESHOLD}
+      archive: true
+      archivedLogFilenamePattern: ${HWX_REGISTRY_LOG_DIR}/${HWX_REGISTRY_LOG_FILE}.%d
+      archivedFileCount: ${HWX_REGISTRY_LOG_FILE_COUNT}
+      timeZone: UTC
+      logFormat: "${HWX_REGISTRY_LOG_FILE_FORMAT}"
```

Comment:
maxFileSize is not working in the version of dropwizard.io bundled with registry.
Once that is done, we